frontend/src/components/CustomApproval/Settings/components/common/utils.ts

```diff
@@ -18,24 +18,39 @@ import {
 } from "@/utils";
 
 export const sourceText = (source: Risk_Source) => {
-  if (source === Risk_Source.SOURCE_UNSPECIFIED) {
-    return t("common.all");
-  }
-
-  const name = risk_SourceToJSON(source);
-  const keypath = `custom-approval.risk-rule.risk.namespace.${name.toLowerCase()}`;
-  if (te(keypath)) {
-    return t(keypath);
+  switch (source) {
+    case Risk_Source.SOURCE_UNSPECIFIED:
+      return t("common.all");
+    case Risk_Source.DDL:
+      return t("custom-approval.risk-rule.risk.namespace.ddl");
+    case Risk_Source.DML:
+      return t("custom-approval.risk-rule.risk.namespace.dml");
+    case Risk_Source.CREATE_DATABASE:
+      return t("custom-approval.risk-rule.risk.namespace.create_database");
+    case Risk_Source.DATA_EXPORT:
+      return t("custom-approval.risk-rule.risk.namespace.data_export");
+    case Risk_Source.REQUEST_QUERY:
+      return t("custom-approval.risk-rule.risk.namespace.request_query");
+    case Risk_Source.REQUEST_EXPORT:
+      return t("custom-approval.risk-rule.risk.namespace.request_export");
+    default:
+      return Risk_Source.UNRECOGNIZED;
   }
-  return name;
 };
 
 export const levelText = (level: number) => {
-  const keypath = `custom-approval.risk-rule.risk.risk-level.${level}`;
-  if (te(keypath)) {
-    return t(keypath);
+  switch (level) {
+    case 0:
+      return t("custom-approval.risk-rule.risk.risk-level.0");
```

Comment:
let's make the `level` more semantic

```suggestion
      return t("custom-approval.risk-rule.risk.risk-level.default");
```

Reply:
Got it, thanks for the suggestion!